src/test/regression.c

```diff
@@ -530,6 +530,150 @@ void test_components()
 	icalerror_set_errors_are_fatal(estate);
 }
 
+static void test_component_foreach_callback(icalcomponent *comp, struct icaltime_span *span, void *data) {
+
+    int* foundExpected = (int*)data;
+    (*foundExpected)++;
+}
+
+int test_component_foreach_parameterized(int startOffsSec, int endOffsSec, int expectedFoundInstances) {
+
+    const char* calStr =
+        "BEGIN:VCALENDAR\n"
+        "BEGIN:VEVENT\n"
+        "DTSTART;20180220T020000Z\n"
+        "DURATION:PT1H\n"
+        "RRULE:FREQ=WEEKLY;BYDAY=TU\n"
+        "END:VEVENT\n"
+        "END:VCALENDAR\n";
+
+    icalcomponent *calendar = icalparser_parse_string(calStr);
+    icalcomponent *event = icalcomponent_get_first_component(calendar, ICAL_VEVENT_COMPONENT);
+
+
+    struct icaltimetype dtstart = icaltime_from_string("20180220T020000Z");
+    struct icaltimetype it_start = dtstart;
+    struct icaltimetype it_end = dtstart;
+	int foundCnt = 0;
```

Comment:
Probably unintended indentation

Reply:
Yes (tab vs spaces), was introduced when last-minute fixing the pedantic compiler errors. Will fix this.